server/controllers/DocumentControllers.js

```diff
@@ -0,0 +1,215 @@
+import pagination from '../helper/pagination';
+
+const Document = require('../models').Document;
```

Comment:
You should be able to do `import {Document} from ''../models'` if you export Document in the models